acceptance/bundle/resources/jobs/update/out.state.direct.json

```diff
@@ -0,0 +1,38 @@
+{
+ "lineage": "[UUID]",
+ "serial": 0,
```

Comment:
Not for this PR: we should begin with 1, 0 might hint at bugs where the field is not initialized.